package.json

```diff
@@ -12,9 +12,8 @@
     "build": "TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\"}' webpack --config webpack.config.ts --mode=development",
     "clean": "rm -rf dist",
     "codecov": "codecov -f .coverage/coverage-final.json",
-    "lint": "tslint --project .",
-    "lint-fix": "tslint --fix --project . && prettier --write './**/*.{js,json}'",
-    "lint:hooks": "eslint \"src/**/*.{ts,tsx}\"",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+    "lint-fix": "eslint . --ext .js,.jsx,.ts,.tsx --fix && prettier --write './**/*.{js,json}'",
```

Comment:
Do we also want prettier to format `ts` and `tsx` files?

Reply:
Good catch. The separate call to prettier is unnecessary. ESLint is configured to use prettier for formatting rules, so `eslint --fix` should catch all of it.
I cleaned up the scripts and `lint-staged` config to skip json/jsx(we don't have jsx files) as well.